src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/resource_allocation_manager_impl.cc

```diff
@@ -23,7 +23,8 @@ ResourceAllocationManagerImpl::ResourceAllocationManagerImpl(
     application_manager::rpc_service::RPCService& rpc_service)
     : current_access_mode_(hmi_apis::Common_RCAccessMode::AUTO_ALLOW)
     , app_mngr_(app_mngr)
-    , rpc_service_(rpc_service) {}
+    , rpc_service_(rpc_service)
+    , is_rc_enabled_(true) {}
```

Comment:
@LitvinenkoIra Why it is `true` by default? May be add value to ini file and read from there? To make it default-configurable without changing our SDL code.

Reply:
@AByzhynar as i understood from https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/RC/rc_enabling_disabling.md
by default, RC functionality is enabled

Reply:
@LitvinenkoIra I agree. But to make it disabled by default - in case if needed : we have to change SDL code, recompile and so on. In case of .ini file value : just change 1 value. Nothing more.

Reply:
@AByzhynar @ilitvinenko there is no need to add additional profile value for it. RC is enabled by default by requirements. In case if we won't to disable RC we always are able to send RPC. I don't see use case of modifying this default value

Reply:
@LuxoftAKutsan No problem.